docs/fundamentals/diagnostics/runtime-method-events.md

```diff
@@ -0,0 +1,396 @@
+---
+title: "Method Runtime Events"
+description: See ETW events that collect information specific to methods, like CLR method events, CLR method marker or CLR method verbose events, and MethodJittingStarted.
+ms.date: "09/27/2020"
+helpviewer_keywords:
+  - "Method events" [.NET Core]"
+  - "ETW, EventPipe, LTTng method events (CoreCLR)"
+ms.assetid: 167a4459-bb6e-476c-9046-7920880f2bb5
+---
+
+# Method ETW Events
```

Comment:
H1 headers need to be unique across all of ms docs. This one is used by other articles.